docs/content/dagster-cloud/developing-testing/environment-variables-and-secrets.mdx

```diff
@@ -0,0 +1,335 @@
+      </td>
+      <td>
+        <ul
+          style={{
+            marginTop: "0px",
+          }}
+        >
+          <li
+            style={{
+              marginTop: "0px",
+            }}
+          >
+            You're using{" "}
+            <a href="/dagster-cloud/deployment/hybrid">Hybrid deployment</a>
+          </li>
+          <li>
+            You want to manage environment variables outside of Dagster Cloud
+          </li>
+        </ul>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Storage and encryption
+
+This may only apply to management using the UI. Will need to confirm once details are finalized.
+
+<TODO></TODO>
+
+### Reserved variables
+
+[Built-in Dagster Cloud environment variables](#built-in-environment-variables) are reserved and are unavailable for use. If used, the following error will be raised: <TODO></TODO>
+
+---
+
+## Managing environment variables
+
+The simplest way to manage environment variables is to use Dagster Cloud's built-in secrets manager, which allows you to create and manage environment variables right in the UI. If you prefer not to use Dagster Cloud's secrets manager, you can use one of [the other methods](#using-alternative-methods) supported for your deployment type.
+
+### Using the Dagster Cloud UI
+
+<Note>
+  <a href="/dagster-cloud/account/managing-users">
+    <strong>Organization</strong>, <strong>Admin</strong>, or{" "}
+    <strong>Editor</strong> permissions
+  </a>{" "}
+  are required to manage environment variables in Dagster Cloud. This includes
+  viewing the value of an existsing environment variable.
+</Note>
+
+Declaring environment variables in the Dagster Cloud UI is supported for both Serverless and Hybrid deployments.
+
+1. Sign into your Dagster Cloud account.
+2. Click the code location you want to add an environment variable to.
+3. Click the **Environment Variables** tab.
+4. Click **+ Add** to add a new environment variable.
+5. In the window that displays, fill in the following:
+   - **Key name** - Enter a name for the environment variable. This is how the variable will be referenced in your code.
+   - **Value** - Enter a value for the environment variable.
+   - Select the deployment(s) to apply the variable to:
+     - **Production**
+     - **Branch Deployments**
+     - **Local**
+6. When finished, click **Save**.
+
+### Using alternative methods
+
+<TabGroup>
+  <TabItem name="dagster-cloud CLI (Serverless)">
+    <Note>Using the CLI to pass environment variables is only available to Serverless deployments.</Note>
+<p>If you're using a git provider like Bitbucket, you can use the <a href="/dagster-cloud/developing-testing/dagster-cloud-cli">dagster-cloud CLI</a> to pass in environment variables with the <code>--env</code> flag.</p>
+
+<p>
+  This flag accepts <code>KEY=VALUE</code> pairs. For example:
+</p>
+
+```shell
+dagster-cloud serverless deploy --env DATABASE_USERNAME=hooli_testing
```

Comment:
as per above we will take this out / replace it with some other CLI method most likely for syncing secrets to the same API that powers the dagster cloud UI